tests/integration/features/command/user-remove.feature

```diff
@@ -8,6 +8,7 @@ Feature: User remove
     Given user "participant1" creates room "room1" (v4)
       | roomType | 1 |
       | invite   | participant2 |
+    And sleep 1
```

Comment:
Okay, after this test is seems like the problem is not creating the rooms, but the room list is sorted by last chat message and that order is mixed up for postgres as there is no sleep between their generation.